src/Extensions.Hosting/HostBuilderExtensions.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Diagnostics;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Json;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+using Serilog;
+
+namespace Bit.Extensions.Hosting;
+
+public static class HostBuilderExtensions
+{
+    /// <summary>
+    ///
+    /// </summary>
+    public static IHostBuilder UseBitwardenDefaults(this IHostBuilder hostBuilder, Action<BitwardenHostOptions>? configure = null)
+    {
+        // We could default to not including logging in development environments like we currently do.
```

Comment:
I'd always log in dev environments and choose a mechanism that's easy to inspect--typically rolling files or stdout, since they're widely compatible. Rationale: logs that you don't see/use while developing typically have lower quality than those you interact with regularly.

Reply:
Oh yeah we for sure want logs for dev environments, the intention with this comment is that we could just not include our custom logging rules/dependencies and leave the ASP.NET Core defaults which would include console and debug providers out of the box, which is what the logging does right now.

Reply:
Ah! I'd say that in the comment, then. Removes the ambiguity between "default logging" and "no logging".